src/lib/utilities/stack-trace/get-file-paths-from-stack-trace.test.ts

```diff
@@ -0,0 +1,86 @@
+import { describe, expect, it } from 'vitest';
+import UnixTSStacktraceQueryJson from '../../../fixtures/stacktrace-query.ts-sdk.json';
```

Comment:
We should be able to update these to `$fixtures/stacktrace-query.ts-sdk.json`.

Reply:
I've tried that. Even though the code compiles and works, VSCode gives a 'Cannot find module' error for this syntaxis.